src/components/instrument-badge/instrument-badge.jsx

```diff
@@ -0,0 +1,130 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import injectSheet from 'react-jss';
+import { Icon } from '../../';
+
+export const styles = theme => {
+  const { palette, typography } = theme;
+
+  return {
+    backgroundCircle: {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      height: 70,
+      width: 70,
+      borderRadius: 35,
+      backgroundColor: palette.color.grayLighter,
+    },
+    badge: {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      width: '100%',
+      height: '100%',
+    },
+    instrumentLvl: {
+      ...typography.secondary,
+      fontSize: ({ size }) => (size === 'sm' ? 12 : 24),
+      width: ({ size }) => (size === 'sm' ? 26 : 43),
+      height: ({ size }) => (size === 'sm' ? 15 : 25),
+      color: palette.color.white,
+      display: 'flex',
+      position: 'absolute',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    wrapper: {
+      ...typography.primary,
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+    },
+    successFail: {
+      backgroundColor: ({ qualified }) => (qualified ? palette.color.green : palette.color.red),
+      color: palette.color.white,
+      position: 'absolute',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      height: ({ size }) => (size === 'sm' ? 13 : 21),
+      width: ({ size }) => (size === 'sm' ? 13 : 21),
+      borderRadius: ({ size }) => (size === 'sm' ? 7 : 12),
+      marginLeft: props => (props.size === 'sm' ? 10 : 16),
+      marginTop: props => (props.size === 'sm' ? 16 : 26),
+    },
+    subText: {
+      color: palette.color.grayDarker,
+      fontSize: ({ size }) => (size === 'sm' ? 10 : 16),
+      textAlign: 'center',
+      paddingTop: 2,
+    },
+  };
+};
+
+const iconProps = (size, strokeWidth, padding) => ({
+  strokeWidth,
+  fill: 'white',
```

Comment:
You can use `fill: 'currentColor', stroke: 'currentColor'` for these instead to be able to control them from the css with color (or you could just set these values in the css directly if they could differ).

Reply:
fixd